interactive.js

```diff
@@ -1,243 +1,184 @@
 import process from 'node:process';
-import chalk from 'chalk';
-import inquirer from 'inquirer';
-import inquirerAutocompletePrompt from 'inquirer-autocomplete-prompt';
-import psList from 'ps-list';
-import {numberSortDescending} from 'num-sort';
+import {createRequire} from 'node:module';
+import React, {useState, useEffect} from 'react';
+import {Box, Text} from 'ink';
 import escExit from 'esc-exit';
-import cliTruncate from 'cli-truncate';
-import {allPortsWithPid} from 'pid-port';
 import fkill from 'fkill';
-import processExists from 'process-exists';
-import FuzzySearch from 'fuzzy-search';
+import hasAnsi from 'has-ansi';
+import {processExited, filterProcesses} from './utils.js';
+import {Dialog} from './dialog.js';
 
-const isWindows = process.platform === 'win32';
-const commandLineMargins = 4;
+const require = createRequire(import.meta.url);
+const TextInput = require('ink-text-input').default;
+const SelectInput = require('ink-select-input').default;
```

Comment:
Use `import`.

Reply:
I'm not sure the reason, but when using `import` on these ink libs, I've encountered the below error message.

```
Check the render method of `InteractiveUI`.
    at InteractiveUI (file:///Users/igyubong/Desktop/open-source/fkill-cli/distribution/interactive.js:20:3)
    at App (/Users/igyubong/Desktop/open-source/fkill-cli/node_modules/ink/build/components/App.js:42:9)

  ERROR  Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object.

        Check the render method of `InteractiveUI`.

 - 
 - Check the render method of `InteractiveUI`.
 - createFiberFromTypeAndProps (node_modules/react-reconciler/cjs/react-reconciler.development.js:17777:21)
 - createFiberFromElement (node_modules/react-reconciler/cjs/react-reconciler.development.js:17805:15)
 - createChild (node_modules/react-reconciler/cjs/react-reconciler.development.js:4827:28)
 - reconcileChildrenArray (node_modules/react-reconciler/cjs/react-reconciler.development.js:5100:25)
 - reconcileChildFibers (node_modules/react-reconciler/cjs/react-reconciler.development.js:5506:14)
 - reconcileChildren (node_modules/react-reconciler/cjs/react-reconciler.development.js:8417:28)
 - updateHostComponent (node_modules/react-reconciler/cjs/react-reconciler.development.js:9059:3)
 - beginWork (node_modules/react-reconciler/cjs/react-reconciler.development.js:10507:14)
 - Object.invokeGuardedCallbackProd (node_modules/react-reconciler/cjs/react-reconciler.development.js:12101:10)
 - invokeGuardedCallback (node_modules/react-reconciler/cjs/react-reconciler.development.js:12292:31)
```